src/ducks/sharing/share.styl

```diff
@@ -0,0 +1,60 @@
+@import 'cozy-ui'
+
+:global
+    label[for='pho-album-share-toggle']
+        cursor pointer
+
+:local
+    .pho-share-modal-content
```

Comment:
Ok, I could understand, just imagine that `pho-` is a acronym for `poor holding object` ;)